Essentials/src/main/java/com/earth2me/essentials/commands/Commandcondense.java

```diff
@@ -153,6 +154,8 @@ private SimpleRecipe getCondenseType(final ItemStack stack) {
             return null;
         }
 
+        final boolean legacy = VersionUtil.getServerBukkitVersion().isLowerThan(VersionUtil.v1_13_0_R01);
```

Comment:
With the sheer amount of VersionUtil calls just to determine whether we're on a pre-flattening or post-flattening version, would it be worth throwing a static `POST_FLATTENING` field on `VersionUtil` for the sake of readability?